index.html

```diff
@@ -1343,42 +1355,50 @@
           </p>
           <p>
             When the <code><dfn>close</dfn>()</code> method is called on a
-            <a>PresentationConnection</a>, the user agent MUST run the
-            algorithm to <a data-lt="close-algorithm">close a presentation
-            connection</a> with <a>PresentationConnection</a>, a
-            <code>reason</code> of <code>closed</code>, and an empty
-            <code>message</code>.
+            <a>PresentationConnection</a> <var>S</var>, the <a>user agent</a>
+            MUST <a>start closing the presentation connection</a> <var>S</var>
+            with <code>closed</code> as <code>closeReason</code> and an empty
+            <code>message</code> as <code>closeMessage</code>.
```

Comment:
The problems that I was trying to prevent in my PR here are:

1. ensure that two consecutive calls to close() won't "start closing the presentation connection" twice, to avoid signaling the other end twice and triggering two "closed" events.
2. ensure that calling close() on a presentation connection on which the closing procedure has already started through other means (object being discarded) would do nothing
3. ensure that the presentation connection state gets switched to "closed" immediately when close() is called, in other words I would expect the code in the `if` block below not to run, ever:
```
presentation.close();
if (presentation.state === "connected") {
  // This should never happen, presentation.state should already be "closed"
}
```

For 3., this would be consistent with WebSockets and RTCDataChannel (well, they use an intermediary "closing" state, but the point is that the presentation should no longer be "connected"). However, here, the "start closing the presentation connection" calls on the "close the presentation connection" procedure that "queues a task" to set the state to "closed". That task will run on next tick by definition. There needs to be some step here or in the closing procedure that instructs the user agent to set the state to "closed" immediately in the close() case.

I'm not entirely sure that 2. is possible, as the only other way to "start closing the presentation connection" is when 
the object is discarded, which might only happen at the very end when there are no longer any references to it. However, it seems to me that this could happen while there is still some JS code to run that holds a reference to the connection.